doc/article/technical_article.tex

```diff
@@ -121,17 +121,19 @@
 
   The backend is written in Python with the help of the web application
   framework Django and the frontend with the Javascript UI framework Vue.js.
-  Both technologies were new to most team members, but have proven themselves
-  effective and learning them were ultimately a benefit to the outcome of the
-  project.
+  Both technologies were new to most team members, but all members have proven
+  themselves
+  %effective
```

Comment:
Why is this commented out? Remove it if not needed anymore

Reply:
the word was marked by the teacher. It's a suggestion for an alternative sentence :)